interfaces/apparmor/backend_test.go

```diff
@@ -341,22 +341,29 @@ var combineSnippetsScenarios = []combineSnippetsScenario{{
 	// Classic confinement uses apparmor in complain mode by default.
 	opts:    interfaces.ConfinementOptions{Classic: true},
 	snippet: "snippet",
-	content: commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected,complain) {\nsnippet\n}\n",
+	content: "\n(classic)" + commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected,complain) {\nsnippet\n}\n",
 }, {
 	// Classic confinement in JailMode uses enforcing apparmor.
 	opts:    interfaces.ConfinementOptions{Classic: true, JailMode: true},
 	snippet: "snippet",
-	content: commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected) {\nsnippet\n}\n",
+	content: "\n(classic)" + commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected) {\nsnippet\n}\n",
```

Comment:
I didn't review the previously already merged PR and just noticed that we apparently support JailMode with classic confinement. This seems to simply toggle 'complain' on the classic template. The template is completely wide open so strict vs complain means nothing and will only confuse users. Instead, please unconditionally add ',complain' to the template and drop jail mode for --classic altogether.